### DISCUSSION TITLE
Discussion: implement a new git handler

### DISCUSSION THREAD

**Discussion by avsm** [mirage/irmin#145] [Ideas]
This may help with integration with watches and hooks:
https://rovaughn.github.io/2015-2-9.html

It's about how to create a new protocol handler (e.g. `git push irmin://github.com`) in the git CLI.

**Comment by AltGr** [comment]
Interesting! Actually, I was going to ask: Irmin has a git backend, but what about a git front-end ? e.g. speaking the git-over-http protocol (either the dumb or the dynamic one) ? Does that exist already ? Would it be difficult ?

Could be quite useful for, I don't know, serving an opam repository for a completely random example :p
